### PR TITLE
Updates for mybinder

### PR DESCRIPTION
**Pull request by haleepfl**

### CODE REVIEW

File: neuror/cut_plane/detection.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def _get_points(neuron):
                      for point in section.points])
 
 
-def plot(neuron, result):
+def plot(neuron, result, inline=False):
     '''Plot the neuron, the cut plane and the cut leaves'''
```
Comment (wizmer) [inline neuror/cut_plane/detection.py]:
```suggestion
    '''Plot the neuron, the cut plane and the cut leaves
    
    Args:
        neuron: the neuron
        result: the cut plane object
        inline: whether or not to plot as an interactive plot (like in a Jupyter notebook)
    '''
```

**Comment by haleepfl** [comment]
With plotly plots, the output will not be shown on github. I can still include it, but it is quite a lot of output, and I'll add it when the review is almost complete to reduce the noise in the PR diff. I'll post a link to the mybinder once I've fixed an issue I'm having.

**Comment by jdcourcol** [comment]
> With plotly plots, the output will not be shown on github. I can still include it, but it is quite a lot of output, and I'll add it when the review is almost complete to reduce the noise in the PR diff. I'll post a link to the mybinder once I've fixed an issue I'm having.

ok. thank you.

**Comment by haleepfl** [comment]
@jdcourcol here is the [mybiner link](https://mybinder.org/v2/gh/BlueBrain/NeuroR/enh/mybinder?filepath=examples%2Fcut-plane-detection.ipynb)

**Pull request merged**